docs/tutorials/dapp/setup.md

```diff
@@ -112,6 +112,18 @@ npm start
 
 You should see the results in the terminal or in the browser dev tool.
 
+### In the terminal
+
+<p align="center">
+  <img src="assets/jssdk_terminal.jpg" width="400">
+</p>
```

Comment:
try this format?  ![avatar](/home/picture/1.png)

Reply:
png or jpg is ok

Reply:
maybe you just need to copy the output, and put into 
```
OUTPUT
```

Reply:
eeee, a good idea.
But image may be more intuitive.